README_ZH-CN.md

```diff
@@ -138,6 +138,7 @@
 * [httpie](https://github.com/jkbrzt/httpie) - HTTPie 是一个命令行 HTTP 客户端，用户友好的 cURL 替代品
 * [ngincat](https://github.com/jaburns/ngincat) - 使用 netcat 的微型 Bash HTTP 服务器
 * [resty](https://github.com/micha/resty) - 你可以在管道中使用的小型命令行 REST 客户端
+* [lrzsz](https://ohse.de/uwe/software/lrzsz.html) - windows下配合xshell进行上传下载操作，简单易操作
```

Comment:
Please keep the section ordered alphabetically (move `lrzsz` up to between `httpie` and `ngincat`).